Mathlib/Data/Finset/Sum.lean

```diff
@@ -163,6 +160,22 @@ lemma disjSum_eq_iff : s.disjSum t = u ↔ s = u.toLeft ∧ t = u.toRight :=
 lemma eq_disjSum_iff : u = s.disjSum t ↔ u.toLeft = s ∧ u.toRight = t :=
   ⟨fun h => by simp [h], fun h => by simp [← h, toLeft_disjSum_toRight]⟩
 
+lemma disjSum_subset : s.disjSum t ⊆ u ↔ s ⊆ u.toLeft ∧ t ⊆ u.toRight := by simp [subset_iff]
+lemma subset_disjSum : u ⊆ s.disjSum t ↔ u.toLeft ⊆ s ∧ u.toRight ⊆ t := by simp [subset_iff]
+
+@[simp] lemma map_disjSum_inl_subset : s.map .inl ⊆ u ↔ s ⊆ u.toLeft := by
+  simp [← disjSum_empty, disjSum_subset]
+
+@[simp] lemma map_disjSum_inr_subset : t.map .inr ⊆ u ↔ t ⊆ u.toRight := by
+  simp [← empty_disjSum, disjSum_subset]
+
+lemma subset_map_disjSum_inl : u ⊆ s.map .inl ↔ u.toLeft ⊆ s ∧ u.toRight = ∅ := by
+  simp [← disjSum_empty, subset_disjSum]
+
+lemma subset_map_disjSum_inr : u ⊆ t.map .inr ↔ u.toLeft = ∅ ∧ u.toRight ⊆ t := by
+  simp [← empty_disjSum, subset_disjSum]
```

Comment:
These are misnamed as they no longer mention `disjSum`.

It's not clear to me that the first ones are stated in the right direction; it feels like `simp`ing `s ⊆ insert x t` to `s.erase x ⊆ t`